eda/data_analysis.py

```diff
@@ -123,164 +214,13 @@ def get_category_sector(index: int = 0) -> Tuple[str, str]:
-        ("Technology Products & Services", "Unassigned"),
-        ("People Services", "Culture, Media and Sport"),
-        ("Technology Solutions & Outcomes", "Culture, Media and Sport"),
-        ("Workforce Health & Education", "Infrastructure"),
-        ("Marcomms & Research", "Education"),
-        ("Construction", "Unassigned"),
-        ("Professional Services", "Culture, Media and Sport"),
-        ("Workforce Health & Education", "Culture, Media and Sport"),
-        ("PSR & Permanent Recruitment", "Infrastructure"),
-        ("Marcomms & Research", "Culture, Media and Sport"),
-        ("Marcomms & Research", "Local Community and Housing"),
-        ("Construction", "Culture, Media and Sport"),
-        ("Workplace", "Culture, Media and Sport"),
-        ("Digital Future", "Unassigned"),
-        ("PSR & Permanent Recruitment", "Local Community and Housing"),
-        ("Travel", "Unassigned"),
-        ("PSR & Permanent Recruitment", "Education"),
-        ("PSR & Permanent Recruitment", "Culture, Media and Sport"),
-        ("Workplace", "Unassigned"),
-        ("Contact Centres", "Government Policy"),
-        ("Technology Solutions & Outcomes", "Unassigned"),
-        ("People Services", "Unassigned"),
-        ("Contact Centres", "Defence and Security"),
-        ("Professional Services", "Unassigned"),
-        ("Marcomms & Research", "Unassigned"),
-        ("Contact Centres", "Health"),
-        ("Financial Planning & Estates", "Government Policy"),
-        ("Contact Centres", "Education"),
-        ("PSR & Permanent Recruitment", "Unassigned"),
-        ("Commodities and Innovation", "Government Policy"),
-        ("Contact Centres", "Infrastructure"),
-        # Below 4 have just 67 records
-        ("Financial Planning & Estates", "Defence and Security"),
-        ("Financial Planning & Estates", "Culture, Media and Sport"),
-        ("Financial Planning & Estates", "Education"),
-        ("Financial Planning & Estates", "Local Community and Housing"),
-        # Data is too less for the below, so we are not using it.
-        # ("Contact Centres", "Local Community and Housing"),
-        # ("Contact Centres", "Unassigned"),
-        # ("Contact Centres", "Culture, Media and Sport"),
-        # ("Managed Service", "Defence and Security"),
-        # ("Financial Planning & Estates", "Unassigned"),
-        # ("Below Threshold", "Health"),
-        # ("Managed Service", "Government Policy"),
-        # ("Managed Service", "Unassigned"),
-        # ("Managed Service", "Infrastructure"),
-        # ("Managed Service", "Education"),
-        # ("Below Threshold", "Defence and Security"),
-        # ("Below Threshold", "Local Community and Housing"),
-        # ("Below Threshold", "Government Policy"),
-        # ("Below Threshold", "Education"),
-        # ("Below Threshold", "Culture, Media and Sport")
-    ]
+
+    if not os.path.exists(local_category_sector_path):
+        get_all_category_sector(latest=True)
```

Comment:
When the local file doesn't exist and get into line 219, then probably the user can't easily predict which combinations they will get by the index number they input. 
Would it be good if we change this to save the result of get_all_category_sector to local file, then ask the user re-run the script after checking the index number with local file?

Reply:
Generally no one remembers all the categories and sectors, so this method is just a helper method that user can use find a combination of category. It doesn't matter much from where it is coming. 99% of the time when data scientist wants to fine tune anything he won't be relying on this method instead he will be using something like this:
category, sector = ("Construction","Education") i.e. passing hard core values to see how particular combination is behaving. So this doesn't require change.